tests/testing_tools.py

```diff
@@ -60,4 +59,17 @@ def generate_random_md5(self):
         md5_hash = hashlib.md5(random_string.encode()).hexdigest()
 
         return md5_hash
+    def paths_generator(self, date: str):
```

Comment:
added function paths_generator to generate the paths variable for initializing the TestPicturaeImporter class in unittests.